examples/usage_example.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import time
+
+from pprint import pprint
+from zapv2 import ZAPv2
+
+target = 'http://127.0.0.1'
+
+# By default ZAP API client will connect to port 8080
+zap = ZAPv2(proxies={'http': '127.0.0.1:9090', 'https': '127.0.0.1:9090'})
```

Comment:
Should specify the proxy in this statement? If so, the comment and following example needs to be updated.